README

```diff
@@ -1,13 +1,78 @@
-This README is for userspace RDMA fabric library.
-
 Version @VERSION@
 Released on @DATE@
 
+Introduction
+============
+The Open Fabrics Interfaces (OFI) is a framework focused on exporting
+fabric communication services to applications.  Libfabric is a software
+library instantiation of OFI.
+
+Libfabric has the following objectives:
+
+* High-performance: provide optimized software paths to hardware
+  - Independent of hardware implementations
+* Scalable: targets support for millions of processes
+  - Designed to reduce cache and memory footprint
+  - Scalable address resolution and storage
+  - Tight data structures
+* Application-centric
+  - Interfaces co-designed with application developers and hardware
+    vendors
+* Extensible
+  - Easily adaptable to support future application needs
+
+OFI is being developed by the OFI Working Group (OFIWG) a subgroup
+of the OpenFabrics Alliance (OFA).  Participation in OFIWG
+(pronounced o-fee-wig) is open to anyone, regardless of their
+membership in OFA.
+
+The goal of OFI and libfabric is to define interfaces that enable
+a tight semantic map between applications and underlying fabric
+services.  Specifically, libfabric software interfaces have been
+co-designed with fabric hardware providers and application developers,
+with a focus on the needs of HPC users.  OFI supports multiple
+interface semantics, is fabric and hardware implementation agnostic,
+and leverages and expands the existing RDMA open source community. 
+
+For more information regarding the OFI project, please visit the OFIWG
+github site:
+
+http://ofiwg.github.io/libfabric/
+
+
+Support
+=======
+OFI targets support for the Linux operating system. A reasonable effort
+is made to support all major, modern Linux distributions; however,
+validation is limited to the most recent 2-3 releases of RedHat
+Enterprise Linux (RHEL)and SUSE Linux Enterprise Server (SLES).
+OFI aligns its supported distributions with the most current
+OpenFabrics Enterprise Distribution (OFED) software releases. With
+the exception of the sockets provider, which is provided for development
+purposes, distro support for a specific provider is vendor specific.
+
+OFI will also run on OS-X, but OS-X support is provided as a convenience
```

Comment:
Also, I think you mean to say "Libfabric will also run on..."   OFI is the framework; libfabric is the implementation.